frontend/webpack.config.ts

```diff
@@ -34,6 +35,11 @@ const virtualModules = new VirtualModulesPlugin();
 const overpassTest = /overpass-.*\.(woff2?|ttf|eot|otf)(\?.*$|$)/;
 const sharedVendorTest = new RegExp(`node_modules\\/(${sharedVendorModules.join('|')})\\/`);
 
+const sharedPatternFlyCoreModules = ['@patternfly/react-core', '@patternfly/react-table'];
+const sharedPatternFlyCore = new RegExp(
+  `node_modules\\/(${sharedPatternFlyCoreModules.map((m) => _.escapeRegExp(m)).join('|')})\\/`,
```

Comment:
Seems like this is a better version of `RegExp` computation for `sharedVendorTest`.

I'd suggest adding a function like this:

```ts
const getVendorModuleRegExp = (vendorModules: string[]) =>
  new RegExp(`node_modules\\/(${vendorModules.map((m) => _.escapeRegExp(m)).join('|')})\\/`);
```

and using it to compute both `sharedVendorTest` and `sharedPatternFlyCoreTest`.